Source/Scene/Scene.js

```diff
@@ -1287,17 +1291,25 @@ define([
             us.updateFrustum(frustum);
             clearDepth.execute(context, passState);
 
-            var commands = frustumCommands.opaqueCommands;
-            var length = frustumCommands.opaqueIndex;
-            for (var j = 0; j < length; ++j) {
-                executeCommand(commands[j], scene, context, passState);
+            var commands;
+            var length;
+
+            // Execute commands in order by pass up to the translucent pass.
+            // Translucent geometry needs special handling (sorting/OIT).
+            var numPasses = Pass.TRANSLUCENT;
```

Comment:
This is a bit shady; it makes assumptions about what is less than `TRANSLUCENT`.  We want our architecture to scale easily when we add shadows, post-process, and cube maps, so we should keep this clean.

At this least, we should explicitly document this in Pass.js.  I would also consider a separate array like:
```
Pass.OpaquePasses = [Pass.GLOBE, Pass.OPAQUE_OBJECTS /* renamed from OPAQUE */];
```